### DISCUSSION TITLE
qMRLab

### DISCUSSION THREAD

**Discussion by zrazi** [qMRLab/qMRLab#478] [General]
Can you use qMRLab for real images versus magnitude images? If so how can I fix the problem of matrices of non-matching size?

**Comment by mathieuboudreau** [comment]
Hi @zrazi - which quantitative model/technique are you trying to use?

**Reply by zrazi** [reply]
Actually, I think my question is more basic, I am stuck when I try to sort my DICOM file and make the required 4D file that meets the requirements. I was fine when I had magnitude images.

**Reply by mathieuboudreau** [reply]
Hi @zrazi, 

Unfortunately at this time qMRLab does not support DICOM images. If you're issue involved converting the DICOMS to NIfTI, which is the more standard file format for neuroimaging and is supported by qMRLab, then I would recommend you open an issue on the dcm2niix project for assistance: https://github.com/rordenlab/dcm2niix 

Thanks!
